### ISSUE TITLE
Discovery complete: 0 families found

### ISSUE THREAD

**Issue by juancrescente** [Dfam-consortium/RepeatModeler#38]
Hello!

I'm getting *Discovery complete: 0 families found*. Seems like in issue in the last step!!

Any ideas?  do I have to run it all over again! :(

```

Processing RECON family: 2222
  - Saving elements to a file...
    - 5 elements found.
Element Gathering: 00:00:00 (hh:mm:ss) Elapsed Time
Refining family-2222 model...
  WARNING: Refiner did not return a consensus.
Refinement: 00:00:00 (hh:mm:ss) Elapsed Time
Family Refinement: 00:00:09 (hh:mm:ss) Elapsed Time
Round Time: 78:58:19 (hh:mm:ss) Elapsed Time

Discovery complete: 0 families found
Program Time: 137:55:13 (hh:mm:ss) Elapsed Time
Working directory:  /home/juan/Desktop/juan/bio/csativa/braker2/RM_4480.TueJun181003232019
may be deleted unless there were problems with the run.
No families identified.  Perhaps the database is too small
or contains overly fragmented sequences.
**
```

Output dir:
```

 l /home/juan/Desktop/juan/bio/csativa/braker2/RM_4480.TueJun181003232019
total 784K
drwxr-xr-x 8 juan juan 4,0K jun 20 21:00 .
drwxr-xr-x 5 juan juan 4,0K jun 18 10:03 ..
-rw-r--r-- 1 juan juan    0 jun 18 10:18 consensi.fa
-rw-r--r-- 1 juan juan    0 jun 18 10:18 families.stk
drwxr-xr-x 2 juan juan 4,0K jun 18 10:14 round-1
drwxr-xr-x 7 juan juan  12K jun 18 10:18 round-2
drwxr-xr-x 7 juan juan  36K jun 18 10:55 round-3
drwxr-xr-x 7 juan juan  76K jun 18 16:40 round-4
drwxr-xr-x 7 juan juan 272K jun 20 21:00 round-5
drwxr-xr-x 7 juan juan 364K jun 24 03:58 round-6

```

**Comment by rmhubley** [comment]
Thanks for reporting this.  Would it be possible to tar/zip up the RM_4480.TueJun181003232019/ directory and send it to me so I can take a look?  rhubley at systemsbiology.org

**Comment by juancrescente** [comment]
This is the content of the dir:
277.462 items, totalling 45,4 GB

du -sh RM_4480.TueJun181003232019 
43G	RM_4480.TueJun181003232019


This is the command I've used

```
BuildDatabase -name PK.db -engine ncbi  PK.fna
RepeatModeler -database PK.db  -engine ncbi -pa 2

```
Where

`-rw-r--r-- 1 juan juan 268M jun 18 14:56 PK.zip
`

**Comment by rmhubley** [comment]
It doesn't look like a problem in the last step only.  The RM_4480.TueJun181003232019/consensi.fa file should have sequences starting after round-1.  That is why I wanted to take a look at various files in each of the round-# directories.  Do you have the full screen output for the run?  Let's start there.

**Comment by juancrescente** [comment]
I do not and it takes several days to execute but I've just started it over and I'm saving the ouput in a file. I'll share it as soon as it finishes

**Comment by juancrescente** [comment]
Hello,

I hope this helps debbuging. Let me know what else can I do.

[out.log](https://github.com/rmhubley/RepeatModeler/files/3351009/out.log)

**Comment by juancrescente** [comment]
any updates on this one @rmhubley ? thanks!

**Comment by WTarabidopsis** [comment]
Hi,
Have you solved the problem? I encountered exactly the same issue when I was running RepeatModeler.
Is there something wrong with the software installation or configuration? I installed RepeatModeler and RepeatMasker from conda with the command `conda install -c bioconda repeatmodeler`
Further, here is a brief summary of my genome, is it overly fragmented?
```
1982 sequences
N50 is 1707373
Total length is 958314969
```
Looking forward to your help, thanks!

**Comment by juancrescente** [comment]
nope, no updates yet

**Comment by rmhubley** [comment]
Really sorry for the delay.  

@juancrescente :  I see from your new log that RepeatModeler is reporting that you have version "RepeatModeler Version DEV".  This isn't a released version of RepeatModeler.  Did you download this from the RepeatMasker website or from the "releases" tab in github?  If not ( e.g. you cloned the master branch or obtained RepeatModeler in someone else's package ), please install it from the sources I list above.  Version 1.0.11 should be what shows up when you run "RepeatModeler -v".  Let's correct this and then I can help you do smaller/faster runs to debug any further problems with this input sequence.

@WTarabidopsis :  You have a different problem.  We didn't create the BioConda recipe for RepeatModeler and we have had lots of reports that it doesn't work.  We have our own BioConda recipe in the works but it's not ready yet.  Please download and install RepeatModeler using the instructions here: http://www.repeatmasker.org/RepeatModeler/

**Comment by WTarabidopsis** [comment]
@rmhubley 
Thanks for the reply, I've re-installed RepeatModeler by following the instructions on the website. However, some new issues came up.
During the running, I got several error messages like this:
```
ERROR from search engine (139)
sh: line 1: 123072 Segmentation fault      (core dumped) /lustre/home/liangjun/01-jiayuxin/00-softwares/RepeatModeler-open-1.0.11/01-dependence/rmblast-2.9.0/rmblastn -db /lustre/home/jiayuxin/4-assembly/4-etuberosom/09-annotation/01-repeatmodeler/05-canu.raw/RM_96838.ThuAug151940422019/round-6/sampleDB-6.fa -query /lustre/home/jiayuxin/4-assembly/4-etuberosom/09-annotation/01-repeatmodeler/05-canu.raw/RM_96838.ThuAug151940422019/round-6/batch-2381.fa -gapopen 20 -gapextend 5 -complexity_adjust -word_size 7 -xdrop_ungap 500 -xdrop_gap_final 250 -xdrop_gap 125 -min_raw_gapped_score 250 -dust no -num_threads 4 -matrix comparison.matrix -gilist /lustre/home/jiayuxin/4-assembly/4-etuberosom/09-annotation/01-repeatmodeler/05-canu.raw/RM_96838.ThuAug151940422019/round-6/batch-2381-gilist 2> /lustre/home/jiayuxin/4-assembly/4-etuberosom/09-annotation/01-repeatmodeler/05-canu.raw/RM_96838.ThuAug151940422019/round-6/ncResults-1565907867-123034.err
```
When I went back to check the `.err` file, it was empty. But finally I could get a result and it looked OK, and I continued to run RepeatMasker to generate a masked genome. 
Could you help me with that problem? I re-installed the dependent softwares and configured for several times but didn't help.

Thanks.

**Comment by jebrosen** [comment]
@WTarabidopsis 
> sh: line 1: 123072 Segmentation fault      (core dumped) (...)/rmblast-2.9.0/rmblastn

rmblast-2.9.0 has a known bug that can result in a Segmentation fault, so we released 2.9.0-p1. You can get it at http://www.repeatmasker.org/RMBlast.html and re-run the `configure` step for RepeatMasker to specify the path to the newer rmblast.

**Comment by WTarabidopsis** [comment]
> @WTarabidopsis
> 
> > sh: line 1: 123072 Segmentation fault      (core dumped) (...)/rmblast-2.9.0/rmblastn
> 
> rmblast-2.9.0 has a known bug that can result in a Segmentation fault, so we released 2.9.0-p1. You can get it at http://www.repeatmasker.org/RMBlast.html and re-run the `configure` step for RepeatMasker to specify the path to the newer rmblast.

Thanks, it worked!

**Comment by Shrihariprasad** [comment]
> @WTarabidopsis
> 
> > sh: line 1: 123072 Segmentation fault      (core dumped) (...)/rmblast-2.9.0/rmblastn
> 
> rmblast-2.9.0 has a known bug that can result in a Segmentation fault, so we released 2.9.0-p1. You can get it at http://www.repeatmasker.org/RMBlast.html and re-run the `configure` step for RepeatMasker to specify the path to the newer rmblast.

My RepeatMasker is configured with rmblast-2.11.0 and while running RepeatClassifier it shows

```
sh: line 1: 169602 Segmentation fault      (core dumped) /var/applications/softwares/RepeatMasker/rmblast-2.11.0/bin/rmblastn -num_alignments 9999999 -db /var/applications/softwares/RepeatMasker/Libraries/RepeatMasker.lib -query tmpConsensi.fa.masked -gapopen 20 -gapextend 5 -complexity_adjust -word_size 7 -xdrop_ungap 500 -xdrop_gap_final 250 -xdrop_gap 125 -min_raw_gapped_score 250 -dust no -num_threads 4 -matrix comparison.matrix 2> ./ncResults-1619765140-168947.err
RepeatClassifier: ERROR from search engine (139) 
Classification Time: 01:06:03 (hh:mm:ss) Elapsed Time
```
Could you help me with the problem? I tried to get the  rmblast-2.9.0-p1but was not able to get the download link

**Comment by Shrihariprasad** [comment]
> @WTarabidopsis
> 
> > sh: line 1: 123072 Segmentation fault      (core dumped) (...)/rmblast-2.9.0/rmblastn
> 
> rmblast-2.9.0 has a known bug that can result in a Segmentation fault, so we released 2.9.0-p1. You can get it at http://www.repeatmasker.org/RMBlast.html and re-run the `configure` step for RepeatMasker to specify the path to the newer rmblast.

My RepeatMasker is configured with rmblast-2.11.0 and while running RepeatClassifier it shows

```
sh: line 1: 169602 Segmentation fault      (core dumped) /var/applications/softwares/RepeatMasker/rmblast-2.11.0/bin/rmblastn -num_alignments 9999999 -db /var/applications/softwares/RepeatMasker/Libraries/RepeatMasker.lib -query tmpConsensi.fa.masked -gapopen 20 -gapextend 5 -complexity_adjust -word_size 7 -xdrop_ungap 500 -xdrop_gap_final 250 -xdrop_gap 125 -min_raw_gapped_score 250 -dust no -num_threads 4 -matrix comparison.matrix 2> ./ncResults-1619765140-168947.err
RepeatClassifier: ERROR from search engine (139) 
Classification Time: 01:06:03 (hh:mm:ss) Elapsed Time
```

Could you help me with the problem? I tried to get the rmblast-2.9.0-p1but was not able to get the download link
